src/media/css/app-list.styl

```diff
@@ -61,6 +62,14 @@ $desktop-tile-padding = 20px 50px;
     text-transform: uppercase;
 }
 
+.app-list.app-list-colored {
+    // When the app list has a colored background (e.g., Feed landing).
+    .mkt-tile h3,
+    .mkt-tile .author {
+        color: $white;
```

Comment:
Not about pre-compiled character count, just good to not use magic colors. And in case we ever wanted to use like off-white (#FFFFFA) instead of pure white, we could change it quickly